sklearn/_loss/tests/test_link.py

```diff
@@ -73,15 +73,15 @@ def test_link_inverse_identity(link):
         # we do not need to distinguish.
         raw_prediction = rng.normal(loc=0, scale=10, size=(n_samples))
 
-    assert_allclose(link.link(link.inverse(raw_prediction)), raw_prediction)
+    assert_allclose(link.link(link.inverse(raw_prediction)), raw_prediction, rtol=1e-2)
```

Comment:
This `rtol` is a bit high. Is it specific for a certain link function?

Reply:
I thinks `raw_prediction = rng.normal(loc=0, scale=10)` sometimes creates very large (or very negative) values. Logit link then takes the exponential and the logarithms which leads to loss of precision.

We can use different random numbers, how about `rng.uniform(-30, 30)`?

Reply:
I think this is worth trying. We should document as well with a comment this behaviour and why we do this choice. This is not a trivial thing.

Reply:
I tried your suggestion but it results in the same test failures compared to using `rng.normal`, i.e. all test failures are related to the `LogitLink` function and the failures occur with the same random seeds.